npm/cypress-schematic/sandbox/package.json

```diff
@@ -10,22 +10,22 @@
     "test": "ng test"
   },
   "dependencies": {
-    "@angular/animations": "~13.1.3",
-    "@angular/common": "~13.1.3",
-    "@angular/compiler": "~13.1.3",
-    "@angular/core": "~13.1.3",
-    "@angular/forms": "~13.1.3",
-    "@angular/platform-browser": "~13.1.3",
-    "@angular/platform-browser-dynamic": "~13.1.3",
-    "@angular/router": "~13.1.3",
+    "@angular/animations": "~12.0.0",
```

Comment:
Not sure why these are changing?

Reply:
Edit Edit: It doesn't look like we want any of the npm/cypress-schematic changes in the PR to go through.

Reply:
oh dang, these didn't even show up as merge conflicts. I wonder if we want to be auto-merging these...

Reply:
I wonder if the original changes went in with a squash and we just lost history there? Definitely something to keep an eye on.